user-console/src/main/java/org/pentaho/mantle/client/ui/tabs/MantleTabPanel.java

```diff
@@ -58,12 +66,75 @@ public class MantleTabPanel extends org.pentaho.gwt.widgets.client.tabs.PentahoT
   private static final String FRAME_ID_PRE = "frame_"; //$NON-NLS-1$
   private static int frameIdCount = 0;
 
+  private static MenuBar tabsMenuBar;
+  private static MenuItem tabsMenuItem;
+  private static TabContextMenuBar tabsSubMenuBar;
+
+  private static final String EMPTY_TABS_MENU = "empty-tabs-menu";
+
+  private HashMap<PentahoTab, MantleTabMenuItem> menuItemHashMap;
+
+  private static class TabContextMenuBar extends com.google.gwt.user.client.ui.MenuBar {
+
+    String height, maxHeight;
+    int left, top, menuBarTop;
+
+    public TabContextMenuBar( boolean b ) {
```

Comment:
`b` is not descriptive, please match "verical" to fall inline with Super class 

https://github.com/gwtproject/gwt/blob/main/user/src/com/google/gwt/user/client/ui/MenuBar.java#L320